src/js/queries/topServicesQuery.js

```diff
@@ -6,6 +6,12 @@ const topServicesQuery = `
           id
           title
           slug
+          coaGlobal
+          departments {
+            id
+            title
+            slug
+          }
```

Comment:
before, it would just break the build if we didn't have a topic for the top services on the home page, something we never noticed because we always get the same 4 and they all have topics